File: app/src/main/java/com/alfianyusufabdullah/chatyuk/ActivityLauncher.java
package com.alfianyusufabdullah.chatyuk;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.google.firebase.auth.FirebaseAuth;


public class ActivityLauncher extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_launcher);

        FirebaseAuth auth = FirebaseAuth.getInstance();

        if (auth.getCurrentUser() != null) {
            startActivity(new Intent(this, ActivityMain.class));
        } else {
            startActivity(new Intent(this, ActivityAuth.class));
        }

        finish();

    }
}

File: app/src/main/java/com/alfianyusufabdullah/chatyuk/ActivityLauncher.kt
package com.alfianyusufabdullah.chatyuk

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import com.google.firebase.auth.FirebaseAuth

class ActivityLauncher : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_launcher)

        val auth = FirebaseAuth.getInstance()

        if (auth.currentUser != null) {
            startActivity(Intent(this, ActivityMain::class.java))
        } else {
            startActivity(Intent(this, ActivityAuth::class.java))
        }

        finish()
    }
}
